scripts/nine-main.js

```diff
@@ -0,0 +1,313 @@
+        let aiFullWin = checkWin(buildBoardState(regBoard), player, undefined);
+        if (aiFullWin === true) {
+            return textUpdate.textContent = `${player === 'x' ? playerXName : playerOName} WINS!!!`;
+        } else {
+            //checks if board being moved to is full
+            if ('finished' in fullBoard[nextPlayerBoard]) {
+                boards.forEach((board) => {
+                    if (!('finished' in fullBoard[boards.indexOf(board)])) {
+                        boardReference[boards.indexOf(board)].forEach((cell) => {
+                            if (fullBoard[boards.indexOf(board)][boardReference[boards.indexOf(board)].indexOf(cell)] === false)
+                                cell.addEventListener('click', clicked)
+                        });
+                        board.style.opacity = '1';
+                        board.style.backgroundColor = '#f0f0f0';
+                    }
+                })
+            } else {
+                boardReference[nextPlayerBoard].forEach((cell) => {
+                    if (fullBoard[nextPlayerBoard][boardReference[nextPlayerBoard].indexOf(cell)] === false)
+                        cell.addEventListener('click', clicked)
+                });
+                boards[nextPlayerBoard].style.opacity = '1';
+                boards[nextPlayerBoard].style.backgroundColor = '#f0f0f0';
+
+                player = player === 'x' ? 'o' : 'x';
+                return textUpdate.textContent = `It's ${player === 'x' ? playerXName : playerOName}'s turn!`
+            }
+        }
+    } else {
+
+        //checks if board being moved to is full
+        if ('finished' in fullBoard[nextPlayerBoard]) {
+            boards.forEach((board) => {
+                if (!('finished' in fullBoard[boards.indexOf(board)])) {
+                    boardReference[boards.indexOf(board)].forEach((cell) => {
+                        if (fullBoard[boards.indexOf(board)][boardReference[boards.indexOf(board)].indexOf(cell)] === false)
+                            cell.addEventListener('click', clicked)
+                    });
+                    board.style.opacity = '1';
+                    board.style.backgroundColor = '#f0f0f0';
+                }
+            })
+            player = player === 'x' ? 'o' : 'x';
+            return textUpdate.textContent = `It's ${player === 'x' ? playerXName : playerOName}'s turn!`
+        } else {
+            //updates boards for next turn
+            boardReference[nextBoard].forEach((cell) => cell.removeEventListener('click', clicked));
+            boards[nextBoard].style.opacity = '.25';
+            boards[nextBoard].style.backgroundColor = 'lightGray';
+            if (postFullBoard === true) {
+                cells.forEach((cell) => cell.removeEventListener('click', clicked));
+                boards.forEach((board)=>{
+                    board.style.opacity = '.25';
+                    board.style.backgroundColor = 'lightGray';
```

Comment:
You could set both opacity and color at the same time if you used an rgba color value e.g. `rgb(75, 75, 75, 0.25)`